libraries/botframework-connector/src/auth/emulatorValidation.ts

```diff
@@ -75,6 +75,15 @@ export namespace EmulatorValidation {
             return false;
         }
 
+        if (Array.isArray(ToBotFromBotOrEmulatorTokenValidationParameters.issuer)) {
```

Comment:
This method is very well documented. We should include some comments explaining what we are evaluating here.

Reply:
same here as mentioned in DotNet's pr

Reply:
@ceciliaavila @sw-joelmut 
feedback applied

Reply:
I think we could add the `tid` like dotnet, just to maintain parity, even though its not being used here `const tenantId = token?.payload?.tid ?? '';` or maybe we could use it like this `const tenantId = token?.payload?.[AuthenticationConstants.TenantIdClaim] ?? '';`

dotnet =>
![image](https://github.com/southworks/botbuilder-js/assets/62260472/0d7fabc6-d9d8-4c24-b5cb-0a691465802e)

Reply:
fixed